features/user_can_buy_concert_tickets.feature

```diff
@@ -0,0 +1,12 @@
+Feature: User can buy concert tickets
+  As a registered user
+  In order to purchase tickets with discount
+  I want to be able to access a ticket sales page.
+
+  Scenario: User can buy tickets
+    Given I am on the index page
+    Then I click the button "Concert tickets"
+    Then I should be on "Ticket sales" page
+    And I should see "Upcomming Concerts"
+    Then I click the "Buy ticket" button
+    Then show me the page
```

Comment:
Remove Then show me the page please 👍

Reply:
And then what? This scenario needs an assertion.